### ISSUE TITLE
Can't get filter work

### ISSUE THREAD

**Issue by HydrelioxGitHub** [maxwroc/battery-state-card#64]
**Describe the bug**
I have several sensor that id ends with '_battery' but they don't auto add in the card using the filter feature

**How to reproduce**
Just add the configuration to the card through HA Lovelace UI

**Expected behavior**
At least ten sensor would be integrated in the card

**YAML configuration**
```yaml
type: 'custom:battery-state-card'
title: Battery levels
entities:
  - sensor.button_table_battery
filter:
  include:
    - name: entity_id
      value: '*battery'

```

**Screenshots**
![image](https://user-images.githubusercontent.com/15862735/82674610-dd6ef600-9c43-11ea-93fe-98fb66a99131.png)

**Version**
1.3.4

**Comment by maxwroc** [comment]
Please take a look at documentation https://github.com/maxwroc/battery-state-card#filter-object

And example: https://github.com/maxwroc/battery-state-card#entity-filtering-and-bulk-renaming

You have there exact case how to match entities based on the entity_id field

**Comment by maxwroc** [comment]
If you still won't be able to figure out, let me know here and I'll try to help

**Comment by HydrelioxGitHub** [comment]
My card setup that I provide in the issue is based on a copy/paste of the documentation. I've only remove the 'level' in the filter that I don't have in my entities I'd.

**Comment by maxwroc** [comment]
Sorry you're right, I haven't read it carefully and closed this issue too fast. Will take a look.

**Comment by maxwroc** [comment]
Ok the example was wrong / not complete. You need to add operator as otherwise it compares the values exactly (`button_table_battery` != `*battery`)

So the following config should work for you:
```yaml
type: 'custom:battery-state-card'
title: Battery levels
entities:
  - sensor.button_table_battery
filter:
  include:
    - name: entity_id
      operator: 'matches'
      value: '*battery'
```

Please let me know in case of problems

**Comment by maxwroc** [comment]
Actually I remember I was adding the code which was auto-setting "matches" when wildcard is used but now I don't see it :) Probably I have accidentally removed it when I was refactoring. I'll re-add it

**Comment by maxwroc** [comment]
No the fault is on my side! The doc states that in case of wildcard it should be "matches" by default. Sorry about that

**Comment by maxwroc** [comment]
Fixed in v1.3.5
